### ISSUE TITLE
ERROR: Could not load classifier

### ISSUE THREAD

**Issue by marhjoh** [RIT-NTNU-Bachelor/face-detection-opencv-cpp#3]
ERROR: Could not load classifier - loading haarcascade. Something is probably wrong with the path.

**Comment by marhjoh** [comment]
**OpenCV**
- [Cascade Classifier](https://docs.opencv.org/3.4/db/d28/tutorial_cascade_classifier.html)

**Stackoverflow**
- [CascadeClassifier in OpenCV generates error](https://stackoverflow.com/questions/24974827/cascadeclassifier-in-opencv-generates-error?rq=3)
- [OpenCV (CvHaarClassifierCascade*) cvLoad doesn't load, unable to load xml file](https://stackoverflow.com/questions/7158039/opencv-cvhaarclassifiercascade-cvload-doesnt-load-unable-to-load-xml-file)

**Other**
- [How to detect human faces in real-time in OpenCV using C++?](https://www.tutorialspoint.com/how-to-detect-human-faces-in-real-time-in-opencv-using-cplusplus)

**Comment by marhjoh** [comment]
Fixed the issue. The issue relied within project properties. I had specified both the debugger lib and the release lib for all configurations which created the issue. 

I fixed it through directly specifing the 451d.lib for debug and the 451.lib for release.  All configurations has nothing specified anymore (see pictures).

![image](https://github.com/RIT-NTNU-Bachelor/face-detection-opencv-cpp/assets/94908112/8fba3f01-50d4-4c7c-8a1e-7b8cda286fd9)
![image](https://github.com/RIT-NTNU-Bachelor/face-detection-opencv-cpp/assets/94908112/4081e072-1cc2-4759-984b-927dd8fa92c0)
![image](https://github.com/RIT-NTNU-Bachelor/face-detection-opencv-cpp/assets/94908112/074a3d70-1b9f-4418-a0d8-e6539969069c)
